sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/ScannerWsClientProvider.java

```diff
@@ -58,7 +68,44 @@ public synchronized ScannerWsClient provide(final GlobalProperties settings, fin
       if (!proxyUser.isEmpty()) {
         connectorBuilder.proxyCredentials(proxyUser, System.getProperty("http.proxyPassword"));
       }
+      if(Boolean.valueOf( System.getProperty( "sonar.http.ssl.insecure", "false" ) )) {
+        try {
+          // Create a trust manager that does not validate certificate chains
+          X509TrustManager easyTrust =
+              new X509TrustManager() {
+                @Override
+                public void checkClientTrusted(X509Certificate[] chain, String authType)
+                    throws CertificateException
+                {
+                  // no op
+                }
+
+                @Override
+                public void checkServerTrusted(X509Certificate[] chain, String authType)
+                    throws CertificateException {
+                  // no op
+                }
+
+                @Override
+                public X509Certificate[] getAcceptedIssuers() {
+                  return new X509Certificate[0];
+                }
+              };
 
+          // Init the ezy trusting trust manager
+          SSLContext sslContext = SSLContext.getInstance( "SSL");
+          sslContext.init( new KeyManager[0], new TrustManager[]{easyTrust}, new java.security.SecureRandom());
+          // get an ssl socket factory with our all trusting manager
+          SSLSocketFactory sslSocketFactory = sslContext.getSocketFactory();
+          connectorBuilder.setSSLSocketFactory( sslSocketFactory ).setTrustManager( easyTrust );
+        } catch ( NoSuchAlgorithmException | KeyManagementException e ) {
+          // should not happen but just in case log it
+          throw new RuntimeException( e.getMessage(), e );
+        }
+      }
+      if(Boolean.valueOf( System.getProperty( "sonar.http.ssl.allowall", "false" ) )) {
```

Comment:
@olamy You can find it here: https://github.com/SonarSource/sonar-developer-toolset

I was mainly picking on `if(Boolean.valueOf( System.getProperty( "sonar.http.ssl.allowall", "false" ) )) {` instead of the default formatting style in both IntelliJ and Eclipse which would be `if (Boolean.valueOf(System.getProperty("sonar.http.ssl.allowall", "false"))) {`. And I see several other cases like this, still in the updated code.